generators/index.js

```diff
@@ -34,7 +34,7 @@ module.exports = yeoman.Base.extend({
         this.props.year = (new Date()).getFullYear();
         this.props.class = 'widget-' + Date.now();
         this.props.safe = props.name.replace(/ /g, '-');
```

Comment:
Can't check usage atm, but I think `safe` should either be removed or have the same replace syntax